src/components/WorkPoolEditForm.vue

```diff
@@ -18,8 +18,17 @@
           <p-number-input :id="id" v-model="concurrencyLimit" placeholder="Unlimited" :min="0" />
         </template>
       </p-label>
+
+      <p-label v-if="can.access.workers" label="Type">
+        <WorkPoolTypeSelect v-model:selected="type" disabled />
+      </p-label>
+
+      <p-label v-if="type && schemaHasProperties && can.access.workers" label="Base Job Configuration">
```

Comment:
```suggestion
      <p-label v-if="type && schemaHasProperties && can.access.workers" label="Base Job Configuration">
```
Can we move the `label` to a localized field? We also generally use `computed` variables in the setup script for more complicated conditionals, so:
```ts
type && schemaHasProperties && can.access.workers
```

would become

```ts
const showSchemaForm = computed(() => type.value && schemaHasProperties.value && can.access.workers)
```